### ISSUE TITLE
Syntax error 😢

### ISSUE THREAD

**Issue by mxstbr** [relekang/micro-analytics-adapter-redis#6]
Running against the current `master` branch of `micro-analytics`:

```
❯ DB_ADAPTER=redis ./cli.js       

> micro-analytics-cli@1.1.0 start /path/micro-analytics
> micro dist/index.js "/usr/bin/node" "/path/micro-analytics/cli.js"

micro: Error when importing /path/micro-analytics/dist/index.js: /path/micro-analytics/node_modules/micro-analytics-adapter-redis/index.js:24

          ),
          ^
SyntaxError: Unexpected token )
```

**Comment by relekang** [comment]
Seems like prettier broke this, will fix :v:

**Comment by relekang** [comment]
Fixed in v0.1.1
